Documentation/ApiOverview/DataHandler/UsingDataHandler/Index.rst

```diff
@@ -145,7 +119,6 @@ Complex data submission
 Imagine the :php:`$data` array contains something like this:
 
 ..  code-block:: php
-    :caption: EXT:my_extension/Classes/MyClass.php
```

Comment:
Note: removing the caption is intended, is the given data structure is not bound to a class. However, this data structure is used in the class example below in the right context.